### PR TITLE
Fixes being able to sell camera mobs to CentCom

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes an issue described in the comments of #20238

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/shuttle/supply.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		/mob/living,
+		/mob/camera,
```
Comment (kevinz000) [inline code/modules/shuttle/supply.dm]: can the ai now block shuttles by looking at them?

Comment (Cyberboss) [reply code/modules/shuttle/supply.dm]: Just cargo, and i doubt an ai would be that persistent.

Comment (kevinz000) [reply code/modules/shuttle/supply.dm]: time to abuse

Comment (XDTM) [reply code/modules/shuttle/supply.dm]: Build a xenobio console there,  keep it stuck forever

**Comment by nicbn** [comment]
Just make the camera unable to stand at shuttle.

**Comment by ShizCalev** [comment]
@nicbn That'd fuck with blobs.

**Comment by nicbn** [comment]
Make blob camera an exception to this.

**Comment by ninjanomnom** [comment]
A better solution to this would be to only sell things which move with the shuttle and stop cameras from being moved by shuttles.

**Comment by optimumtact** [comment]
Ninjanomnom is right

**Pull request merged**